project/Settings.scala

```diff
@@ -166,6 +167,7 @@ object Settings {
     val UnboundId              = "com.unboundid"             %   "unboundid-ldapsdk-minimal-edition" % LibraryVersions.unboundId
     val JwtCore                = "com.pauldijou"             %%  "jwt-core"                          % LibraryVersions.jwt
     val Slf4j                  = "org.slf4j"                 %   "slf4j-api"                         % LibraryVersions.slf4j
+    val Slf4jSimple            = "org.slf4j"                 %   "slf4j-simple"                      % LibraryVersions.slf4j
```

Comment:
Why this?

Reply:
http4s logging was getting discarded because there was no log handler on the classpath. We can swap in a better one if we want to, I just went for the easy one.

Reply:
This is a bit strange, we do have logging from http4s at the moment. I wonder what changed

Reply:
I think it was in the gem launcher, not the seqexec one. I'll take a closer look.

Reply:
I removed this and added the `Logging` settings you're using in seqexec.